doc/production/README.md

```diff
@@ -0,0 +1,43 @@
+# Production
+
+Nowadays, you can run JavaScript code in several different environments, some of them with their own particularities. Moreover, you can use `js-libp2p` for a wide range of use cases. Different environments and different use cases mean different configurations and challenges in the network.
+
+Libp2p nodes can vary from nodes behind an application, to infrastructure nodes that enable the network to operate and to be efficient. In this context, the Libp2p project provides public infrastructure to boost the network, enable nodes connectivity and improve constrained nodes performance. This public infrastructure should be leveraged for learning the concepts and experimenting. When an application on top of libp2p aims to move into production, its own infrastructure should be setup as the public nodes will be intensively used by others and its availability is not guaranteed.
+
+This guide aims to guide you from using the public infrastructure into setting up your own.
+
+## Table of Contents
+
+* [Production](#production)
+  * [Star servers](#star-servers)
+  * [Delegate nodes](#delegate-nodes)
+  * [Circuit Relay](#circuit-relay)
+  * [SSL](#ssl)
+
+## `webrtc-star` servers
```

Comment:
I think it would be helpful to more clearly state what environments these are useful for, which is to dial a browser. If that's not really in my user flow, I don't need them.

Also listing alternatives would be helpful. In this instance, relays.